jar-service/pom.xml

```diff
@@ -200,8 +200,8 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<target>1.6</target>
-					<source>1.6</source>
+					<target>1.7</target>
+					<source>1.7</source>
```

Comment:
Ok. I did a grep into the whole code and I only found 
```xml
<source>1.6</source>
```
in `SlipStreamServer/jar-service/pom.xml` 
and in `SlipStreamServer/static-content/target/results/META-INF/maven/org.clojure/clojure/pom.xml`